### DISCUSSION TITLE
Make a firmware release with the old PS encoder

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#420] [Ideas]
I apologize for contacting you again on this topic.  I tested the old and new firmware on a long trip.  Alas, the PS encoder also rotates the PS line poorly with errors when RDS reception is weakened.  It just got off to a good start, but practically nothing else has changed.  The old firmware 1.21 performed well in this regard.  As I understand it, there are no more options to fix it, so I am asking for the opportunity to release firmware with the old PS encoder from old firmware up to 2.0. Is this possible?  Thank you:)

**Comment by PE5PVB** [comment]
I can't revert to another version. The principe is the same as with that version, there are only a lot of extra decoders added. Possibly it's also a timing problem in the RDS signal.
